openedx/core/lib/js_utils.py

```diff
@@ -68,3 +52,32 @@ def escape_json_dumps(obj, cls=EdxJSONEncoder):
     encoded_json = json.dumps(obj, ensure_ascii=True, cls=cls)
     encoded_json = _escape_json_for_html(encoded_json)
     return encoded_json
+
+
+def escape_js_str(js_str):
```

Comment:
This is a replacement for Django's escapejs filter.  Due to its usage (see below), I personally prefer something shorter.

Unfortunately, although "escape_javascript" is almost as short, I personally think it is too vague.  

Here are some options:
1. escape_js_str (leave as-is)
2. escape_js_string (remove one of two abbreviations)
3. escape_javascript_string (no abbreviations)

I vote for #1, then #2.
@andy-armstrong, @cahrens, @nedbat - do you want to vote on your preference?